video_lipsyncing/utils.py

```diff
@@ -124,6 +124,8 @@ def predict(self, face, audio_file, outfile, resize_factor, faces, start_frame):
        
         video_stream = cv2.VideoCapture(face)
         fps = video_stream.get(cv2.CAP_PROP_FPS)
+
+        os.mkdir('temp') #Make the temp directory in case it doesn't exist
```

Comment:
this will error if the function is re-ran because the directory already exists. need to add check.